test/java/org/opendatakit/briefcase/export/ExportToCsvScenario.java

```diff
@@ -0,0 +1,295 @@
+        assertThat(readAllBytes(newMediaPath.resolve(oldMediaPath.relativize(path))), equalTo(readAllBytes(path)))
+    );
+  }
+
+  void assertSameContentRepeats(String... groupNames) {
+    Arrays.asList(groupNames).forEach(groupName -> {
+      String oldOutput = new String(readAllBytes(outputDir.resolve("old").resolve(stripIllegalChars(formDef.getFormName()) + "_" + groupName + ".csv")));
+      String newOutput = new String(readAllBytes(outputDir.resolve("new").resolve(stripIllegalChars(formDef.getFormName()) + "-" + groupName + ".csv")));
+      assertThat(newOutput, is(oldOutput));
+    });
+  }
+
+  Path getSubmissionDir() {
+    return formDir.resolve("instances").resolve(instanceID.orElseThrow(RuntimeException::new).replaceAll(":", ""));
+  }
+
+  void createInstance(String submissionTpl, LocalDate submissionDate) {
+    createInstance(submissionTpl, submissionDate, "some value");
+  }
+
+  void createInstance(String submissionTpl, LocalDate submissionDate, String value) {
+    String formattedSubmissionDate = submissionDate.atStartOfDay().atOffset(OffsetDateTime.now().getOffset()).format(ISO_OFFSET_DATE_TIME);
+    String instanceId = "uuid" + UUID.randomUUID();
+    String instanceContent = String.format(submissionTpl, instanceId, formattedSubmissionDate, formattedSubmissionDate, value);
+    Path instanceDir = formDir.resolve("instances").resolve(instanceId);
+    createDirectories(instanceDir);
+    Path instanceFile = instanceDir.resolve("submission.xml");
+    write(instanceFile, instanceContent.getBytes());
+  }
+
+  void createOutputFile(String dir, String file) {
+    createDirectories(outputDir.resolve("old").resolve(dir));
+    write(outputDir.resolve("old").resolve(dir).resolve(file), "Some content".getBytes());
+    createDirectories(outputDir.resolve("new").resolve(dir));
+    write(outputDir.resolve("new").resolve(dir).resolve(file), "Some content".getBytes());
+  }
+
+  void assertNoOutputMediaDir() {
+    assertThat(Files.exists(outputDir.resolve("old").resolve("media")), is(false));
+    assertThat(Files.exists(outputDir.resolve("new").resolve("media")), is(false));
+  }
+
+  String readFile(String fileName) {
+    return new String(readAllBytes(getPath(fileName)));
+  }
+
+  static Path getPath(String fileName) {
+    return maybeGetPath(fileName).orElseThrow(RuntimeException::new);
+  }
+
+  private static Optional<Path> maybeGetPath(String fileName) {
+    return Optional.ofNullable(ExportToCsvTest.class.getClassLoader().getResource("org/opendatakit/briefcase/export/" + fileName))
+        .map(url -> Paths.get(toURI(url)));
+  }
+
+  private static Optional<String> readInstanceId(String formName) {
+    return maybeGetPath(formName + "_submission.xml")
+        .filter(Files::exists)
+        .map(ExportToCsvScenario::readInstanceId);
+  }
+
+  private static String readInstanceId(Path path) {
+    return readAllLines(path).get(0).split("instanceID=\"")[1].split("\"")[0];
```

Comment:
Keep in mind that some forms are gigantic. Why not just read the first line?
```
    return readOneLine(path).split("instanceID=\"")[1].split("\"")[0];
```
![screen shot 2018-05-13 at 1 03 45 pm](https://user-images.githubusercontent.com/55099/39971343-1eaf5926-56ae-11e8-8ec7-8087b3b0e4d5.png)